src/D1_model_components.py

```diff
@@ -372,7 +372,7 @@ def source_non_dispatchable_fix(model, dict_asset, **kwargs):
                 actual_value=dict_asset["timeseries"],
                 fixed=True,
                 nominal_value=dict_asset["installedCap"]["value"],
-                variable_costs=dict_asset["opex_var"][0],
+                variable_costs=dict_asset["opex_var"]["value"][index],
```

Comment:
```suggestion
                variable_costs=dict_asset["opex_var"][0],
```
Actually not correct, nut keeps merge order intact